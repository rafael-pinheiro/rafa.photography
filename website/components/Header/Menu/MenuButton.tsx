import styles from './MenuButton.module.scss';

export default function MenuButton({ onClick } : { onClick: () => void}) {
  return (
    <button
      className = {styles.button}
      onClick = {onClick}
    >
      <span className={styles.label}>menu</span>
      <span className={styles.hamburguer}>
        <span className={styles.line} />
        <span className={styles.line} />
        <span className={styles.line} />
      </span>
    </button>
  )
}
