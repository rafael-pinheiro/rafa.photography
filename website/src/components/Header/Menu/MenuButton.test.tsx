import renderer from 'react-test-renderer';
import MenuButton from './MenuButton';

it('renders correctly', () => {
  const tree = renderer
    .create(<MenuButton />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
