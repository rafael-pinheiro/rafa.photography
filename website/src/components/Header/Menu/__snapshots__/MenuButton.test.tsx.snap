// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button>
  <span>
    menu
  </span>
  <span>
    <span />
    <span />
    <span />
  </span>
</button>
`;
